homeassistant/components/zamg/weather.py

```diff
@@ -80,6 +87,12 @@ def setup_platform(
 class ZamgWeather(WeatherEntity):
     """Representation of a weather condition."""
 
+    _attr_native_pressure_unit = (
+        LENGTH_MILLIMETERS  # API reports l/m², equivalent to mm
```

Comment:
mm is not a valid pressure unit.

https://github.com/home-assistant/core/blob/ef76073d8336717c865e2560076f2b0379d2f456/homeassistant/util/pressure.py#L20-L30

Should this be a precipitation unit?